examples/Baselines/Halite_competition/torch/submission.py

```diff
@@ -0,0 +1,986 @@
+#   Copyright (c) 2021 PaddlePaddle Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import pickle
+import base64
+import random
+import numpy as np
+
+import torch
+import torch.nn as nn
+import torch.nn.functional as F
+from torch.distributions import Categorical
+from kaggle_environments.envs.halite.helpers import *
+from kaggle_environments import make, evaluate
+from collections import deque
+
```

Comment:
When use `parl` with torch, if the torch model extends from `parl.model`, it is recommended to add 
`import os; os.environ['PARL_BACKEND'] = 'torch'` for the reason that if the python env has pip the paddle and torch in the same, maybe the `parl` will use the `paddle` for default.